### PR TITLE
Update sponsor logos

### PR DESCRIPTION
**Pull request by RobertWSaunders**
## Proposed Change

Updates logos to include new sponsors and reworks some responsiveness stuff.

### How did you do this?

Add the new logos and CSS!

### Why are you choosing this approach?

Make sure to represent sponsors on our website!

### Any open questions you want to ask code reviewers?

Nope!

### List of changes:

  - Add National Bank logo
  - Update Distribute Compute Labs logo
  - Mobile responsiveness tweaks

## Pull Request Checklist:

  - [x] My submission passes all tests.
  - [x] I have lint my code locally prior to submission.
  - [x] I have written new unit tests for my core changes (where applicable).
  - [x] I have written browser integration tests for my core changes (where application).
  - [x] I have referenced all useful information (issues, tasks, etc) in the pull request.

### CODE REVIEW

**Comment by RobertWSaunders** [comment]
Deploy preview for *qhacks-website* ready!

Built with commit 7baa4d093215e72db7f899881db2ac5e5fc3a72b

https://deploy-preview-215--qhacks-website.netlify.com

File: src/assets/img/sponsors/nationalbank.svg [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (joeytepp) [inline src/assets/img/sponsors/nationalbank.svg]: This is 4 spaces 😞

Comment (joeytepp) [inline src/assets/img/sponsors/nationalbank.svg]: This is 4 spaces 😞

File: src/assets/img/sponsors/dcl.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...3L1491.23,392Zm-61-39.23h44.09l-22.05-51.55Z"/><path class="cls-6" d="M1583.53,345.57q5.16,6.84,5.17,16.88,0,14-9.66,21.74t-27,7.76h-46.83V283.69h45.31q16.73,0,26.08,7.45T1586,312a24.52,24.52,0,0,1-17,24.17Q1578.35,338.74,1583.53,345.57Zm-62.8-14.9h27.37q22.65,0,22.65-17.48,0-16.89-22.65-16.88h-27.37ZM1567.87,375q5.48-4.33,5.47-13.45,0-9.29-5.55-13.76t-17.26-4.49h-29.8v36h29.8Q1562.4,379.33,1567.87,375Z"/><path class="cls-6" d="M1621.77,389.82a50.06,50.06,0,0,1-18-10l5.62-11.55a53.54,53.54,0,0,0,16.43,9.27,59.25,59.25,0,0,0,19.15,2.89q12.16,0,18.86-4.56a14.73,14.73,0,0,0,6.69-12.92,11.47,11.47,0,0,0-6.24-10.57q-6.23-3.57-19.76-6.62a111.56,111.56,0,0,1-21.14-6.31,32.5,32.5,0,0,1-12.92-9.57q-4.71-6-4.71-15.44a28.51,28.51,0,0,1,5-16.65A33,33,0,0,1,1625,286.5a52.52,52.52,0,0,1,21.21-4,64.06,64.06,0,0,1,21.36,3.5,45.68,45.68,0,0,1,16.65,9.88l-5.62,11.56a53.1,53.1,0,0,0-15.36-9.2,47.61,47.61,0,0,0-16.88-3q-11.7,0-18.4,4.95a15.83,15.83,0,0,0-6.69,13.45,12.41,12.41,0,0,0,5.86,11.1q5.85,3.81,18.78,6.84a148.1,148.1,0,0,1,21.89,6.39,34.11,34.11,0,0,1,13.38,9.12q5,5.79,5,14.9a27.38,27.38,0,0,1-5,16.42,32.35,32.35,0,0,1-14.37,10.88q-9.35,3.87-22,3.87A75,75,0,0,1,1621.77,389.82Z"/></g></g></svg>
```
Comment (joeytepp) [inline src/assets/img/sponsors/dcl.svg]: This should be formatted like the rest of our SVGs

**Pull request merged**